client/wfnews-war/src/main/angular/src/app/utils/error-messages.ts

```diff
@@ -0,0 +1,43 @@
+import {ValidationError} from "../store/application/application.state";
+
+const digits = () => `must contain only numbers`;
+const required = () => `is required`;
+const invalid = () => `is invalid`;
+const notFound = () => `not found`;
+const min = ([message, min]: any[]) => `cannot be less than ${min}`;
+const max = ([message, max]: any[]) => `cannot be more than ${max}`;
+const minMax = ([message, min, max]: any[]) => `must be between ${min} and ${max} characters`;
+
+export const ErrorMessages = {
+    "required": () => "Value is required",
+    "max": (x) => `Value cannot be more than ${x}`,
+    "min": (x) => `Value cannot be less than ${x}`,
+    "maxlength": (x) => `Value cannot exceed ${x} characters`,
+    "minlength": (x) => `Value must be at least ${x} characters`,
+    "signup": () => "Please sign up before proceeding",
+    "notsignedup": () => "Employee is currently not signed up",
+
+    "error.employee.number.invalid": () => `The provided employee number doesn't match our records.`,
```

Comment:
What is all this for? We wont have these error messages so no sense in polluting our code with it